arrow-syntax/build.gradle

```diff
@@ -17,6 +17,6 @@ dependencies {
     compile project(":arrow-annotations")
     compile project(":arrow-core")
     testRuntime "org.junit.vintage:junit-vintage-engine:$JUNIT_VINTAGE_VERSION"
-    testCompile "io.kotlintest:kotlintest-runner-junit5:$KOTLIN_TEST_VERSION", excludeArrow
+    testCompile "io.kotest:kotest-runner-junit5-jvm:$KOTLIN_TEST_VERSION", excludeArrow
```

Comment:
I think it's just necessary to remove `excludeArrow` here because `testCompile` dependency.